docs/content/how-to/openstack/create-openstack-cluster.md

```diff
@@ -192,6 +193,129 @@ NAME      VERSION       AVAILABLE   PROGRESSING   SINCE   STATUS
 version   4.17.0        True        False         5m39s   Cluster version is 4.17.0
 ```
 
+## Enabling Ingress
+
+Note: In 4.17, enabling Ingress is a day 2 operation and we tested it with MetalLB. This might change in the future.
+
+* Create a floating IP in your OpenStack cloud that will be the public IP for Ingress.
+* Update your DNS to create the wildcard for *apps.<cluster-name>.<base-domain> .
+* Create the OpenStack Neutron port for Ingress:
+
+```shell
+openstack port create --network k8s-clusterapi-cluster-<cluster-namespace>-<cluster-name>-<cluster-id> --fixed-ip ip-address=<internal-ingress-IP> hcp-ingress
+```
+
+Note: if CAPO handles the network resources, the internal ingress IP should be `10.0.0.7`.
```

Comment:
do we require this address because we're enforcing usage of one CIDR for the machines? is there a chance this won't be always true even when capo handles creation of resources?

Reply:
in the current code as you know we use the `10.0.0.0/16` subnet if not provided by the user so I think it's mention here a valid IP that can be used for Ingress. I could maybe clarify that?

Reply:
yes, I think it's a good idea to clarify it

Reply:
I clarified it. Let me know what you think.